publisher.rb

```diff
@@ -0,0 +1,38 @@
+require 'aws-sdk-sqs'
+class Publisher
+	# Note: You can store the values of the aws access keys in a config file (json) 
+	# from where you can get the values into the constants.
+	ACCESS_KEY_ID = #<your access key id>
+	SECRET_ACCESS_KEY = #<your secret access key>
+
+	attr_accessor :sqs 
+	def initialize
+		creds = Aws::Credentials.new(ACCESS_KEY_ID, SECRET_ACCESS_KEY)
+		@sqs = Aws::SQS::Client.new(region: 'ap-south-1', credentials: creds)
+	end
+
+	# For Standard Queue
+	def create_std_q queue_name
+		std_queue = sqs.create_queue(queue_name: queue_name)
+	end
+
+	# For FIFO Queue
+	def create_fifo_q queue_name
+		queue_name = queue_name.split('.')[0]
+		fifo_queue = sqs.create_queue(queue_name: "#{queue_name}.fifo", attributes: { "FifoQueue": "true" })
+	end
+
+	def publish_messages queue_url
+		999.times do |count|
+			message = { count: "%03d" % count, timestamp: Time.now.strftime("%GW%V%uT%H%M%S%L%z")}.to_json 
+			puts "Message generated - #{message}"
+			puts "Sending message..."
+			sqs.send_message(queue_url: queue_url, message_body: message, delay_seconds: 2)
```

Comment:
A message has to be sent every 5 seconds (make it configurable) - does this line do that?

Reply:
@gnoronha-mdsol yes this line does that. So the message delay as of now is of 2 seconds. Will change it to 5.

Reply:
done. Made it configurable and defaulting it to 5 if nothing is specified.